src/Sarif.Viewer.VisualStudio.UnitTests/ImportanceToForegroundConverterTests .cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I'm confused. Why does this file show up as new code, when @kschecht already merged his changes?